test/expected/build/application.b/standalone/resources/sap-ui-custom.js

```diff
@@ -5,10 +5,10 @@ jQuery.sap.registerPreloadedModules({
 	"id1/embedded/i18n/i18n_de.properties":'title=embedded-i18n_de',
 	"id1/embedded/i18n_fr.properties":'title=embedded-i18n_fr-wrong',
 	"id1/embedded/manifest.json":'{"_version":"1.1.0","sap.app":{"_version":"1.1.0","id":"id1.embedded","type":"component","applicationVersion":{"version":"1.2.2"},"embeddedBy":"../","title":"{{title}}"}}',
-	"id1/i18n.properties":'title=app-i18n-wrong',
-	"id1/i18n/i18n.properties":'title=app-i18n',
-	"id1/i18n/i18n_de.properties":'title=app-i18n_de',
-	"id1/i18n/l10n.properties":'title=app-i18n-wrong',
+	"id1/i18n.properties":'title=app-i18n-wrong\nfame=Stra\\u00dfe',
```

Comment:
This test still seems strange to me. Aren't the input files encoded with different encodings? (i18n.properties vs. i18n/i18n.properties)
How would this work here?

Reply:
it uses the builder
via writePreloadFunction -> writePreloadModule
and reads them as latin1 and then perform the escaping in
lib/lbt/bundle/Builder.js line 448
so this is valid imho